src/server/endpoints/covidcast.py

```diff
@@ -561,6 +561,26 @@ def handle_geo_coverage():
 
     return execute_query(q.query, q.params, fields_string, [], [])
 
+@bp.route("/indicator_geo_coverage", methods=("GET", "POST"))
+def handle_indicator_geo_coverage():
+    source_signal_sets = parse_source_signal_sets()
+    source_signal_sets = restrict_by_roles(source_signal_sets)
+    source_signal_sets, alias_mapper = create_source_signal_alias_mapper(source_signal_sets)
```

Comment:
since the mapper isnt used, we should do this:
```suggestion
    source_signal_sets, _ = create_source_signal_alias_mapper(source_signal_sets)
```